### PR TITLE
Assignment-1

### PR DESCRIPTION
**Pull request by isttiiak**
What is Git Stash? Concept and code.
git stash is a command that temporarily shelves (or stashes) changes you’ve made to your working directory, allowing you to switch branches or perform other tasks without committing the changes to your repository. You can then reapply the stashed changes later.

Git Stash is used in:
Temporary Save: Quickly save unfinished work without committing.
Switch Contexts: Move to another task or branch without losing your current changes.
Clean Working Directory: Get a clean working directory for new tasks or testing.

'git stash'
This saves your changes and reverts your working directory to the last commit. Your changes are now safely stored in the stash list.

git stash apply
This reapplies the most recent stash to your working directory. The stash remains in the stash list.

git stash pop
This reapplies the most recent stash to your working directory and removes it from the stash list.

Use Cases of Stashes:
Interruptions: You’re working on a feature and need to switch branches to address a critical bug.
Experiments: Try out an idea without committing the changes.
Cleanup: Clean your working directory for code reviews or tests without losing your current work.

In a nutshell, git stash is a handy tool for managing our work-in-progress without cluttering our commit history. It provides flexibility to save and retrieve changes easily, allowing for efficient context switching and maintaining a clean working environment.

### CODE REVIEW

**Comment by obijzbo** [comment]
Remove file named "my task" from the project structure and recreate pull request again
No screenshots added in the markdown file
